scripts/1-fetch/fetch_wikicommons.py

```diff
@@ -0,0 +1,343 @@
+#!/usr/bin/env python
+"""
+Fetch CC Legal Tool usage data from WikiCommons API.
+"""
+
+# Standard library
+import argparse
+import csv
+import os
+import sys
+import textwrap
+import traceback
+import urllib.parse
+
+# Third-party
+import requests
+from pygments import highlight
+from pygments.formatters import TerminalFormatter
+from pygments.lexers import PythonTracebackLexer
+from requests.adapters import HTTPAdapter
+from urllib3.util.retry import Retry
+
+# Add parent directory so shared can be imported
+sys.path.append(os.path.join(os.path.dirname(__file__), ".."))
+
+# First-party/Local
+import shared  # noqa: E402
+
+# Setup
+LOGGER, PATHS = shared.setup(__file__)
+
+# Constants
+BASE_URL = "https://commons.wikimedia.org/w/api.php"
+FILE1_COUNT = shared.path_join(PATHS["data_phase"], "wikicommons_1_count.csv")
+HEADER1_COUNT = ["LICENSE", "FILE_COUNT", "PAGE_COUNT"]
+QUARTER = os.path.basename(PATHS["data_quarter"])
+
+# Creative Commons license categories to query
+CC_LICENSE_CATEGORIES = [
```

Comment:
Hi @najuna-brian!

You should sort your constants in alphabetical order. The `CC_LICENSE_CATEGORIES` constant, although voluminous, begins with a `C`